src/Workspaces/VisualBasic/Portable/LanguageServices/VisualBasicSemanticFactsService.vb

```diff
@@ -255,5 +255,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                     Return DirectCast(n.GetSyntax(cancellationToken), TypeStatementSyntax).Modifiers.Any(SyntaxKind.PartialKeyword)
                 End Function)
         End Function
+
+        Public Function IsPossibleTupleContext(semanticModel As SemanticModel, position As Integer, cancellationToken As CancellationToken) As Boolean Implements ISemanticFactsService.IsPossibleTupleContext
+            Throw New NotImplementedException()
```

Comment:
I hadn't had any problem with snippets getting triggered when typing tuples in VB. I didn't investigate in details, but the logic in SnippetCompletionProvider does seem different in VB and C#.